### ISSUE TITLE
[BUG] Authentication failing when running command outside the root folder

### ISSUE THREAD

**Issue by XxUnkn0wnxX** [izu-co/funimation-downloader-nx#5]
** Main info: **

Script version: 4.8.1

** Additional Info: **

Full Log
```
node ~/Apps/Animie/izu-co/funimation-downloader-nx/funi.js --auth

=== Funimation Downloader NX 4.8.1 ===

[INFO] Token not set!

[Q] LOGIN/EMAIL: HIDDEN
[Q] PASSWORD   : HIDDEN
[INFO] Authentication success, your token: 85ffa5a3********************************

node:fs:583
  handleErrorFromBinding(ctx);
  ^

Error: ENOENT: no such file or directory, open 'config/token.yml'
    at Object.openSync (node:fs:583:3)
    at Object.writeFileSync (node:fs:2144:35)
    at auth (/Users/ADMIN/Apps/Animie/izu-co/funimation-downloader-nx/funi.js:102:16)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: 'config/token.yml'
}
```

I have an alias set up for the CLI downloader, this does not occur on the official branch

however when running `funi.js` outside the root folder, so running it like this 
```
node node ~/Apps/Animie/izu-co/funimation-downloader-nx/funi.js --auth
```

causes auth to fail, it fails to write to the config folder...
this did not occur with the last commits. (last working commit: 116a820c32c35acc95dec891486133d154008326)
commit 810d8c703b7cf44d3e350b17ed3f38d7c5731111 TO b2fb0e7260e19e9e32e11b67f2dc3752474fd8de you cannot even call up auth.
did you make a change that made it look for the config folder from where the command was run from?

this issue does not occur when running the command within the root folder only when calling it via the full path outside the root folder...

**Comment by izu-co** [comment]
Hey there. I will take a look at it tomorrow.

**Comment by izu-co** [comment]
Resolved in 3aa4c414986226fecab85f28ede6b4303f2d2116.

As always thanks for the report, and sorry for caused inconveniences!
